tests/tests_pytorch/trainer/logging_/test_logger_connector.py

```diff
@@ -187,11 +187,6 @@ def __init__(self, not_supported):
             {
                 "log",
                 "log_dict",
-                # the following are problematic as they do have `self._current_fx_name` defined some times but
-                # not others depending on where they were called. So we cannot reliably `self.log` in them
```

Comment:
How has this been addressed? Or was it long outdated?

Reply:
the batch transfer hooks are not called using `trainer._call_lightningmodule_hook()`, so they will have current_fx_name and no logging is allowed in these hooks as per `logger_connector/fx_validator`.

Reply:
Okay, I wanted to make sure because the comment must have had a meaning at some point :)